src/GraphQL/GraphQL.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net6</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net7;net6</TargetFrameworks>
```

Comment:
Please remove .NET 7 TFM until we have a ifdef that requires it.

Reply:
Removed.